test/common/common/regex_test.cc

```diff
@@ -13,22 +13,6 @@ namespace Envoy {
 namespace Regex {
 namespace {
 
-TEST(Utility, ParseStdRegex) {
-  EXPECT_THROW_WITH_REGEX(Utility::parseStdRegex("(+invalid)"), EnvoyException,
```

Comment:
I looked at tag_producer_test.cc and did not see a test for the invalid-regex case -- can we drop this in there in this PR?

Reply:
ah was looking in the wrong spot. Thanks!